ui/guest/src/controls/rte.ts

```diff
@@ -152,7 +158,19 @@ export function initTinyMCE(
           e.preventDefault();
         }
       });
-    }
-  });
+    },
+    // Autosave options are left as undefined since it is not supported in control.
+    autosave_ask_before_unload: undefined,
+    autosave_interval: undefined,
+    autosave_prefix: undefined,
+    autosave_restore_when_empty: undefined,
+    autosave_retention: undefined,
+    // No file picker is set by default, and functions are not supported in state.
+    file_picker_callback: undefined,
+    // Height is set to the size of content
+    height: undefined
+  };
+
+  window.tinymce.init(settings);
```

Comment:
What is settings var useful for? Why not leave with the single line init i.e. `init({ ourNonOverridableSettings, ...rteConfig, ourNonOverridableSettings })`?